client/web/compose/src/views/Admin/Modules/Edit.vue

```diff
@@ -301,6 +301,7 @@
                           :is-duplicate="!!duplicateFields[index]"
                           @edit="handleFieldEdit(module.fields[index])"
                           @delete="module.fields.splice(index, 1)"
+                          @updateKind="handleFieldKind(module.fields[index])"
```

Comment:
update based on index and splice same a handleFieldSave, also rename handleFieldKind to handleFieldKindUpdate